src/router.js

```diff
@@ -162,8 +163,12 @@ const router = createRouter({
         {
           path: 'pool/:id',
           name: 'PacbioPoolCreate',
-          component: PacbioPoolCreate,
-          props: true,
+          component: FlaggedFeatureView,
+          props: {
+            feature: 'multiplexing_phase_2_aliquot',
```

Comment:
I would use a different feature flag to this because this one is already in prod and we dont want to turn off existing working features if this one needs to be toggled.